### ISSUE TITLE
Fishing idle notification doesn't play when another fullscreen app has focus

### ISSUE THREAD

**Issue by rdp1408** [runelite/runelite#12566]
Please check if your issue is not a duplicate by [searching existing issues](https://github.com/runelite/runelite/search?type=Issues)

**Describe the bug**
The fishing idle notification does not play a sound when another fullscreen application has focus. I have runelite on a separate monitor and am playing FF:XIV on my main monitor in borderless windowed mode. When I go idle during fishing the idle notification sound does not play. I have verified that the cooking idle notification does work as intended leading me to believe this is a bug and not a configuration issue.

**To Reproduce**
1. Start fishing
2. Click to another fullscreen app/game so it has focus
3. When your character goes idle the fishing notification sound does not play

**Expected behavior**
Fishing notification sound should play just like the Cooking idle notification does.

**Screenshots**

**Environment (please complete the following information):**
 - OS: Windows
 - RuneLite version: 1.6.27
 - Launcher version: 1.6.0

**Additional context**
Add any other context about the problem here, e.g. logs. Your client logs can usually be found in
`C:\Users\<your_user_name>\.runelite\logs` on Windows and `~/.runelite/logs` on Linux and macOS.

**Comment by Nightfirecat** [comment]
This is likely a Windows bug, as we simply dispatch a notification for Windows to display. Can you confirm that you can receive other notifications while FF:XIV is open in fullscreen display?

**Comment by PISTOLCUPCAKES** [comment]
Yes all other notifications work. This happens with other games too. It's just fishing notifications that are borked

**Comment by LlemonDuck** [comment]
You mention you are using an application in full-screen. Windows has an option to automatically mute notifications when an application is in fullscreen, have you checked whether this is enabled?

![image](https://user-images.githubusercontent.com/1868974/105555449-6c527780-5cd7-11eb-901f-beb16698650c.png)
